lib/diagnostic.js

```diff
@@ -6,20 +6,23 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (words) {
+  let normalize = words.map(words.replace(/\W+/,'').toUpperCase());
+  return normalize;
 };
 
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
-const uniqueWords = function (/* */) {
-  /* your response here */
+const uniqueWords = function (words) {
+  words.forEach(normalizeWords(word)) {
+  }
+  return
 };
 
```

Comment:
return what? Try not to leave return statements blank. If anything just return `true` or a 'success' string to later change to what you want.